XLPandXeLaTeX.ltx

```diff
@@ -46,74 +46,69 @@ producing complex documents in beautiful formats. \TeX
-That is, authors cannot input \XeLaTeX\ code directly into the document and the introduction of layout sections within the document flow is constrained via the DTD.
+That is, first, authors cannot input \XeLaTeX\ code directly into the document and second, the introduction of layout sections within the document flow is constrained via the DTD.
 
-\begin{figure*}
-\includegraphics[scale=.5]{Several-different-document-types.png}
+% I took the astrix out following the labe "figure" below in the figure class because that was what I read online which is supposed to make the figure fit only in one column. Centering the figure within the column is still desired but I don't know the stylesheet well enough to know if that is allowed.
+\begin{figure}
+\includegraphics[scale=.3]{Several-different-document-types.png}
 \caption{XLingPaper predefined document types via DTD}
-\end{figure*}
+\end{figure}
 
 \section{What is XLingPaper?}
 
 As mentioned above, XLingPaper is an \XML- and Java-based computer
-plug-in for the XMLmind XML Editor.  It is designed to make writing,
+plug-in for the \XMLmind \XML Editor.  It is designed to make writing,
 reading, and publishing linguistic papers, grammars, and books better
-and more consistent. XLingPaper can produce linguistic documents with
+and more consistent. A full list of benefits to all parties in the publishing workflow is available.\cite{black:2017} XLingPaper can produce linguistic documents with
 at least five outputs, all from the same source document: \PDF{} (version 1.5), Web
 pages (HTML 4), Microsoft Word (.doc), Open Office Writer (.odt), and ePUB. It automatically
-numbers sections, figures, tables, and examples. It keeps track of internal references to these entites along with citation references and gloss abbreviations.  This keeps numbering and reference links dependable and automated. It also
+numbers sections, figures, tables, and examples. It keeps track of internal references to these entites along with citation references, and glosses/abbreviations keeping numbering and reference links dependable and automated. It also
 automatically generates abbreviations used and references cited (using a custom references implimentation).
 
-Unlike most editing programs which are based on either the WYSIWYG paradigm or as text editors used to code or produce markdown, XLingPaper (via the XMLmind XML Editor)
-is a structured editor. Rather than making the document look the way
+Unlike most editing programs which are based on either the \WYSIWYG paradigm or as text editors used to code or produce markdown, XLingPaper (via the \XMLmind \XML Editor)
+is a structured editor. Rather than visually structuring the document look the way
 it is to be formatted, the author “marks up” the items in the document
-according to their kind.  One of the many values this gives is that
+according to their styling needs.  One of the many values this gives is that
```

Comment:
No, I want "kind" here.  It is the object, not the styling that counts.

Reply:
How about using the concept here of `element type`?

Reply:
I still prefer "kind."